.travis/test.sh

```diff
@@ -47,17 +53,15 @@ if [[ ${TASK} == "pylint" ]]; then
 fi
 
 if [[ ${TASK} == "if-else" ]]; then
-    conda create -q -n test-env python=$PYTHON_VERSION numpy
-    source activate test-env
+    pip install numpy
```

Comment:
using conda install should be fast ?

Reply:
I've made it for the uniformity because `rstcheck` isn't available through `conda install`.
Ok, I'll change the installation for all packages that are available through `conda install`.